models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md

```diff
@@ -2,15 +2,19 @@
 
 ## Use Case and High-Level Description
 
-The `efficientnet-b0-pytorch` model is one of the [EfficientNet](https://arxiv.org/abs/1905.11946) group of models designed to perform image classification. This model was pretrained in PyTorch\*. All the EfficientNet models have been pretrained on the ImageNet image database. For details about this family of models, check out the [repository](https://github.com/rwightman/gen-efficientnet-pytorch).
+The `efficientnet-b0-pytorch` model is one of the [EfficientNet](https://arxiv.org/abs/1905.11946) models designed to perform image classification. This model was pretrained in PyTorch\*. All the EfficientNet models have been pretrained on the ImageNet\* image database. For details about this family of models, check out the [TensorFlow EfficientNet repository](https://github.com/rwightman/gen-efficientnet-pytorch).
 
-The model input is a blob that consists of a single image 3x224x224 in RGB
-order. The RGB mean values need to be subtracted as follows: [123.675,116.28,103.53]
-before passing the image blob into the network. In addition, values must be divided
-by [58.395,57.12,57.375].
+### Input 
+
+The model input is a blob that consists of a single image with the [3x224x224] shape in the RGB
+order. Before passing the image blob to the network, do the following:
+1. Subtract the RGB mean values as follows: [123.675,116.28,103.53]
+2. Divide the RGB mean values by  [58.395,57.12,57.375]
+
+### Output 
```

Comment:
Please, leave this section in `Use Case and High-Level Description`, because it part of description

Reply:
It is a children element of  `Use Case and High-Level Description`, just like the `Input` section.

Reply:
Please, leave original structure of `Use Case...` section, because it can be used for internal usage.

Reply:
okay, fixed